backbone.js

```diff
@@ -70,6 +70,22 @@
 
   // Proxy Underscore methods to a Backbone class' prototype using a
   // particular attribute as the data argument
+  var modelMatcher = function(attrs) {
+    return function(model) {
+      return _.matches(attrs)(model.attributes);
```

Comment:
You can pull `_.matches` into the outer scope to save recreating the function all the time:

```js
var modelMatcher = function(attrs) {
  var matcher = _.matches(attrs);
  return function(model) {
    return matcher(model.attributes);
  };
};

```